src/components/docs/Buckets/PrimaryIconBucket/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import BaseContainer from '../BaseContainer';
+
+import styles from './styles.module.css';
+
+const PrimaryIconBucket = ({ description, icon, title, to }) => (
+  <BaseContainer className={styles.root} to={to}>
+    <div className={styles.circleOverlay} />
+    <div className={styles.contents}>
+      <img className={s.image} src={useBaseUrl(icon)} />
```

Comment:
Does this work? Think it should be `styles.image`

Reply:
Whoops. This must have happened when I was commiting. The next commit removes these in favor of background image